lib/ngtsc/workspace.symbols.ts

```diff
@@ -193,6 +194,14 @@ export class WorkspaceSymbols {
       } else if (isToken(token.node)) {
         return this.providerRegistry.getProvider(token.node);
       }
+    } else if (token instanceof ExternalExpr) {
+      const dir = dirname(relativeTo);
+      const module = token.value.moduleName ?? '';
+      const moduleName = module.endsWith('.ts') ? module : `${module}.ts`;
+      const path = join(dir, moduleName);
+      return this.getAllInjectable().filter(injectable => {
```

Comment:
I'm not sure to understand why you need the last one ? 
I've got the feeling that `this.getAllInjectable().find(...)` would be more suited for that.